nilearn/experimental/surface/_surface_image.py

```diff
@@ -14,7 +14,7 @@
 
 
 class Mesh(abc.ABC):
-    """A surface mesh, having vertex coordinates and faces(triangles)."""
+    """A surface mesh, having vertex coordinates and faces (triangles)."""
```

Comment:
```suggestion
    """A surface mesh having vertex, coordinates and faces (triangles)."""
```

Reply:
why not link the terms here too?

Reply:
I missed it. Was working late last night. 🙈